bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncAdmin.java

```diff
@@ -119,6 +120,8 @@ public BigtableAsyncAdmin(CommonConnection asyncConnection) throws IOException {
     if (clusterId != null) {
       bigtableSnapshotClusterName = bigtableInstanceName.toClusterName(clusterId);
     }
+    this.ttlSeconds =
+        configuration.getInt(BigtableOptionsFactory.BIGTABLE_SNAPSHOT_DEFAULT_TTL_SECS_KEY, 86400);
```

Comment:
This should be a shared constant and not duplicated. BigtableOptionsFactory might be a good place for it